.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "relayer/ethashproof"]
+	path = relayer/ethashproof
+	url = git@github.com:Snowfork/ethashproof.git
```

Comment:
Is there reason we need to make ethashproof a submodule? Can't we just list it as a dependency in go.mod, like `github.com/Snowfork/ethashproof`? Then everywhere we import `github.com/tranvictor/ethashproof` we would import `github.com/Snowfork/ethashproof` instead?

Reply:
ye agree seems better

Reply:
No reason we _have_ to make it a submodule. I got the idea from https://github.com/KyberNetwork/bridge_eos_smart_contracts. I think that the monorepo-like situation might benefit us. Two main reasons for this:
* We might want to make changes to ethashproof down the line (definitely for https://github.com/Snowfork/polkadot-ethereum/issues/169), and those changes will have to be in lockstep with the relayer.
* I'm using bits of ethashproof that I would classify as "internals", not the public API. I might reuse bits for transaction proofs too. So managing ethashproof as an internal module is fitting in that sense.

I haven't used submodules before. So I could very well be missing an important consideration. I'm kinda curious to see how this unfolds but will defer to you folks

Reply:
Clarification: `github.com/tranvictor/ethashproof` is the module name declared in ethashproof's go.mod file. So I'm assuming I'd need to update ethashproof's go.mod if I want to import it using `github.com/Snowfork/ethashproof`. That's independent of whether ethashproof is a submodule. I wanted to avoid making changes to ethashproof for this PR but I'll update in another PR